cmd/kubeops/main.go

```diff
@@ -38,6 +39,9 @@ var (
 	userAgentComment       string
 	namespaceHeaderName    string
 	namespaceHeaderPattern string
+	// This version var is updated during the build (see the -ldflags option
+	// in the cmd/kubeops/Dockerfile)
+	version = "devel"
```

Comment:
Thanks for the comment here, I wasn't aware of what the -ldflags was doing actually

Reply:
Yeah, I had to dig when checking how it was being updated for the images, so thought I'd save myself time when I look again in a year :)